cli/tests/pcluster/aws/dummy_aws_api.py

```diff
@@ -208,8 +209,14 @@ def get_filesystem_info(self, fsx_fs_id):
             },
         }
 
+    def set_non_happy(self, non_happy):
```

Comment:
`non_happy` is too generic. How could someone understand that this setting refers specifically to the describe volume API call?

Reply:
It is currently only being used in the describe volume call, but I will rename this more descriptively so it also clear without any knowledge of the implementation.